.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+
+on: [push, pull_request]
+
+env:
+  DISPLAY: :99
+  IS_PULLREQUEST: ${{ github.event.pull_request }}
```

Comment:
When this is used, and I make a PR, I get `Error: .github/workflows/test.yml (Line: 6, Col: 19): A mapping was not expected`

https://github.com/andy5995/appimage.github.io/runs/3002416025?check_suite_focus=true#step:1:25

Reply:
When I make a PR, `${{ github.event.pull_request }}` is substituted for "Object"; if not a PR, the length of the string is zero.

https://github.com/andy5995/appimage.github.io/runs/3002504967?check_suite_focus=true